libbeat/docs/breaking-7.4.asciidoc

```diff
@@ -0,0 +1,47 @@
+[[breaking-changes-7.4]]
+
+=== Breaking changes in 7.4
+
+This section discusses the main changes that you need to be aware of to
+migrate Beats to version 7.4. {see-relnotes}
+
+//NOTE: The notable-breaking-changes tagged regions are re-used in the
+//Installation and Upgrade Guide
+
+//tag::notable-breaking-changes[]
+
+[float]
+==== Golang version update
+
+The Beats platform has been updated to use Golang 1.12.7.
+
+// REVIEWERS: Is the above statement correct (taken from the relnotes)? The 
+// libbeat/docs/version.asciidoc file has the golang version set to 1.12.9.
+
+[float]
+==== {auditbeat} `system` module changes
+
+The `socket` dataset in the `system` module now uses Kprobes for finer-grained
+monitoring and UDP support. For more information, see
+{auditbeat-ref}/auditbeat-dataset-system-socket.html[System socket dataset].
+
+[float]
+==== {filebeat} field name changes
+
+Some field names exported by the `asa` fileset in the `cisco` module have
+changed:
+
+[options="header"]
+|====
+|Old field                 |New field
+|`log.original`            |`event.original`
+|`cisco.asa.list_id`       |`cisco.asa.rule_name`
+|====
+
+
+//REVIEWERS: Do we want to mention the functionbeat deployment/pkg changes here?
```

Comment:
@mikemadden42 Here's another question. I'm assuming the answer is "no" based on an email thread I saw go by.

Reply:
@mikemadden42 I'm going to go ahead and merge this PR (as soon as the CI is built) so the content will be available tomorrow. We can still make changes/additions if necessary. Just let me know.